api4/role.go

```diff
@@ -52,6 +52,13 @@ func getRolesByNames(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	for _, rolename := range rolenames {
+		if !model.IsValidRoleName(rolename) {
+			c.SetInvalidParam("rolename")
```

Comment:
This could potentially be a little frustrating or annoying to an API user if they provide a list of roles `somerole1 somerole2 badrole1 somerole3`. They get a Invalid Param response, but now they need to figure out which role is causing the issue themself

If we just pass down the bad role as well I think it would be much nicer on the API user in case of input error

Non-blocking comment by the way